client/lib/Screens/login/login_screen.dart

```diff
@@ -0,0 +1,145 @@
+import 'package:flutter/material.dart';
+
+class LoginScreen extends StatelessWidget {
+  const LoginScreen({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: ListView(
```

Comment:
Using a ListView for only two items is not recommanded.
Instead you should a SingleChildScrollView + Flex (which can be horizontal/vertical according to the space available)

Reply:
Please show me your code without the `ListView` please.
It should work without any issue here

Reply:
![image](https://user-images.githubusercontent.com/96618001/176872322-dd0654cc-10fd-40ab-b0b9-657058768ee8.png)